controllers/apply/lib/joi-extensions/date-parts.js

```diff
@@ -143,39 +142,6 @@ module.exports = function dateParts(joi) {
                         );
                     }
                 }
-            },
-            {
```

Comment:
Worked out we can do this validation with less specific `minDate` and `maxDate` rules, so if this seems sensible might extract parts of this out into a separate PR that can be merged earlier.